Project.toml

```diff
@@ -26,12 +27,13 @@ Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 SymEngine = "123dc426-2d89-5057-bbad-38513e3affd8"
 Tensors = "48a634ad-e948-5137-8d70-aa71f2a747f4"
 VoronoiDelaunay = "72f80fcb-8c52-57d9-aff0-40c1a3526986"
-ProgressMeter = "92933f4c-e287-5a05-a399-4b506db050ca"
 
 [extras]
 BenchmarkTools = "6e4b80f9-dd63-53aa-95a3-0cdb28fa8baf"
+DiffEqDevTools = "f3b72e0c-5b89-59e1-b016-84e28bfd966d"
+DiffEqOperators = "9fdde737-9c7f-55bf-ade8-46b3f136cc48"
```

Comment:
Sorry, I mixed it up. `DiffEqOperators` is `import`ed in the main package, so this line should be moved to the normal dependencies above. I assume that we don't have to load it in the travis script, so the last change can be undone.

Reply:
Done.